Notify-Tech-Docs/source/documentation/python.md

```diff
@@ -69,338 +109,304 @@ If you are using the [test API key](/#test), all your messages will come back as
 
 All successfully delivered messages will appear on your dashboard.
 
-### API Reference
-
-#### Error
+### Error codes
 
 If the request is not successful, the client will raise an `HTTPError`:
 
 |`error.status_code`|`error.message`|Notes|
 |:---|:---|:---|
-|`429`|`[{`<br>`"error": "RateLimitError",`<br>`"message": "Exceeded rate limit for key type TEAM/TEST/LIVE of 3000 requests per 60 seconds"`<br>`}]`||
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Can't send to this recipient using a team-only API key"`<br>`]}`|Use the correct type of API key, refer to [API keys](/#api-keys) for more information|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Can't send to this recipient when service is in trial mode - see https://www.notifications.service.gov.uk/trial-mode"`<br>`}]`|Refer to [trial mode](https://www.notifications.service.gov.uk/features/using-notify#trial-mode) for more information|
+|`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Error: Your system clock must be accurate to within 30 seconds"`<br>`}]`|Check your system clock|
+|`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Invalid token: signature, api token not found"`<br>`}]`|Check your API key, refer to [API keys](/#api-keys) for more information|
+|`429`|`[{`<br>`"error": "RateLimitError",`<br>`"message": "Exceeded rate limit for key type TEAM/TEST/LIVE of 3000 requests per 60 seconds"`<br>`}]`|Refer to [API rate limits](/#api-rate-limits) for more information|
 |`429`|`[{`<br>`"error": "TooManyRequestsError",`<br>`"message": "Exceeded send limits (LIMIT NUMBER) for today"`<br>`}]`|Refer to [service limits](/#service-limits) for the limit number|
-|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Can"t send to this recipient using a team-only API key"`<br>`]}`||
-|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Can"t send to this recipient when service is in trial mode - see https://www.notifications.service.gov.uk/trial-mode"`<br>`}]`||
+|`500`|`[{`<br>`"error": "Exception",`<br>`"message": "Internal server error"`<br>`}]`|Notify was unable to process the request, resend your notification.|
 
-#### Arguments
-
-##### Required
+## Send an email
 
-`phone_number`
+### Method
 
-The phone number of the recipient of the text message.
+```python
+response = notifications_client.send_email_notification(
+    email_address='sender@something.com', # required string
+    template_id='f33517ff-2a88-4f6e-b855-c550268ce08a', # required UUID string
+)
+```
 
-`template_id`
+### Arguments
 
-Find this by logging into GOV.UK Notify and going to the Templates page.
+#### email_address (required)
 
-##### Optional
+The email address of the recipient, only required for email notifications.
```

Comment:
The implication being that `email_address` is optional for non-email notifications? I’d lose this second clause, don’t think it makes sense or is necessary.